assets/src/edit-story/types.js

```diff
@@ -73,6 +73,7 @@ StoryPropTypes.elements = {};
 StoryPropTypes.elements.image = PropTypes.shape({
   ...StoryElementPropsTypes,
   src: PropTypes.string.isRequired,
+  opacity: PropTypes.number,
```

Comment:
I believe, the `opacity` should go on the `element`. It's a shared property, whether we want to expose it to the each type of element or not.